src/runvm-coreos-installer

```diff
@@ -56,4 +64,11 @@ workdir=$(pwd)
 mkdir -p "${deploydir}/${workdir}/"
 mount --bind "${workdir}" "${deploydir}/${workdir}"
 
+# echo back the invocation
+set -x
+
 RUST_BACKTRACE=full chroot "${deploydir}" env -C "${workdir}" coreos-installer "$@"
+
+if [ -f /var/tmp/coreos-installer-output ]; then
```

Comment:
maybe a comment above this block to help explain what is going on?

Reply:
Done!